asgs.ttl

```diff
@@ -1554,10 +1566,14 @@ asgs:Feature rdf:type owl:Class ;
                              [ rdf:type owl:Restriction ;
                                owl:onProperty asgs:contains ;
                                owl:allValuesFrom asgs:Feature
+                             ] ,
+                             [ rdf:type owl:Restriction ;
```

Comment:
ok, so this was missing until you put this in. i guess the ontologies don't assert that asgs:contains can be an inverse relationship to asgs:within, so we'll have to add it in here.

Reply:
`asgs:within` didn't exist until cf003e211272b5ae1f4a0cfbe0837bf4f3e5e3c4 
I also added 
```
asgs:within rdfs:subPropertyOf geo:sfWithin . 
asgs:contains rdfs:subPropertyOf geo:sfContains .
```
but these are not declared as inverses in GeoSPARQL either. Not sure why - I might ping Matt Perry and see if he can say.